sites/hashdev/src/_pages/blog/10_error-stack-update-0-2.mdx

```diff
@@ -0,0 +1,185 @@
+---
+authorPhoto: blog/authors/tim-diekmann.jpg
+postPhoto: blog/10_error-stack-update-0-2/Cover.jpg
+title: "Announcing error-stack v0.2"
+subtitle: "Redesigned and customizable outputs, related errors and multiple sources, and much more"
+author: "Tim Diekmann"
+jobTitle: "Platform Engineer, HASH"
+date: "2022-10-03"
+---
+
+Back in June [we announced the first version](https://hash.dev/blog/announcing-error-stack) of `error-stack`, our context-aware error library. We developed this to help address a lot of challenges we encountered when implementing error handling in our simulation engine [LINK]. In the time since then we’ve started using it in our entity-graph query layer [LINK?] that we’re building as a datastore for HASH [LINK?]. Across that development we’ve learned a lot of lessons, and in joint efforts with open-source contributors we’re ready to release a new version of the library.
+
+Almost every week since our last release, we’ve been fixing user-reported bugs, reviewing open-source contributor’s pull requests, and adding new features. [Version 0.2](https://github.com/hashintel/hash/tree/error-stack%400.2.0/packages/libs/error-stack) is available now and includes important changes from `std`, a completely redesigned output with new customization options, support for related errors and multiple error sources, and a number of other improvements.
+
+## Changes from `std` in v0.2
+
+- The `Provider` API has landed and replaces our vendored-in implementation.
+- `Backtrace` was stabilized with Rust 1.65(🎉), we now support `Backtrace`s on a non-nightly channel starting with 1.65-beta.
+- If the root context already contains a backtrace, no new `Backtrace` is captured. Otherwise, backtraces are captured automatically when `RUST_BACKTRACE`.
+  or `RUST_LIB_BACKTRACE` env-vars are set.
```

Comment:
```suggestion
- If the root context already contains a backtrace, no new `Backtrace` is captured. Otherwise, backtraces are captured automatically when `RUST_BACKTRACE` or `RUST_LIB_BACKTRACE` env-vars are set.
```